onnxruntime/test/python/onnxruntime_test_python.py

```diff
@@ -69,20 +69,35 @@ def testSetProvidersWithOptions(self):
             options = sess.get_provider_options()
             option = options['TensorrtExecutionProvider']
             self.assertIn('device_id', option)
-            self.assertIn('has_trt_options', option)
+            self.assertIn('trt_max_partition_iterations', option)
```

Comment:
let's limit the scope of this options test to ones we can actually test instead of lighting up every option and not being able to verify them.